src/test/java/edu/ucla/library/iiif/fester/verticles/ManifestVerticleTest.java

```diff
@@ -91,6 +119,39 @@ public void tearDown(final TestContext aContext) {
         myVertx.close(aContext.asyncAssertSuccess());
     }
 
+    /**
+     * Test against the Sinai works.
+     *
+     * @param aContext A testing context
+     */
+    @Test
+    public final void testSinaiWorksManifest(final TestContext aContext) {
+        final String jsonFile = myJsonFiles + "/ark%3A%2F21198%2Fz16t1r0h.json"; // work manifest
+        final String path = StringUtils.format(SINAI_WORKS_CSV, SINAI);
+        final JsonObject message = new JsonObject();
+        final Async asyncTask = aContext.async();
+
+        message.put(Constants.CSV_FILE_NAME, myRunID).put(Constants.CSV_FILE_PATH, path);
+        message.put(Constants.FESTER_HOST, MANIFEST_HOST);
+
+        LOGGER.debug(MessageCodes.MFS_120, SINAI, ManifestVerticle.class.getName());
+
+        myVertx.eventBus().request(ManifestVerticle.class.getName(), message, request -> {
+            if (request.succeeded()) {
+                final JsonObject manifest = new JsonObject(myVertx.fileSystem().readFileBlocking(jsonFile));
+
+                assertEquals(ViewingHint.Option.PAGED.toString(), manifest.getString("viewingHint"));
+                assertEquals(ViewingDirection.RIGHT_TO_LEFT.toString(), manifest.getString("viewingDirection"));
```

Comment:
Should these string literals be sourced from the jiiify-presentation constants, or would that be too verbose?

Reply:
The viewingHint and viewingDirection? We really only use the literals in the tests when they are repeated in the test (because our Checkstyle rules complain if we repeat a string more than once). We could put them in a TestConstants (at this point not sure I see a reason to put them into Constants since they're just used in a test) but since they're not repeated, I'd hold off until we have a need to do this.

Reply:
:+1: